src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  padding: 10px;
+  max-width: calc(7 * #{$day-size} + 2 * #{$calendar-padding} + 7 * #{$day-border-width});
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  flex-wrap: wrap;
+
+  &__day {
+    margin-right: 1px;
```

Comment:
Instruction says to use "gap" to create a distance between cells

Reply:
But mentors say margin))